### DISCUSSION TITLE
Module compatibility with Magento EE 2.4.5-p10

### DISCUSSION THREAD

**Discussion by aswin-eglobe** [Smile-SA/elasticsuite#3474] [Q&A] [answered]
I am using Smile Elastic suite version 2.10.x. I would like to know if this version supports Magento EE 2.4.5-p10 version.

**Comment by rbayet** [comment, marked as answer]
Hello @aswin-eglobe,

Our CI runs checks on EE and CE 2.4.5-p8 but I don't see any reason in EE 2.4.5-p9 and EE 2.4.5-p10 release notes to believe it would not.
Considering 2.4.5-p10 has been published in late October, people would have created issues about some incompatibility by now.

Regards,

**Reply by aswin-eglobe** [reply]
Hi @rbayet 
Thank you for the information.
